README.md

```diff
@@ -1,28 +1,45 @@
 
-Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
+The Kubeflow makes AI/ML on Kubernetes simple, portable and scalable.
 
 ---
 
 ## Documentation
+
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
 
-## Working Groups
-The Kubeflow community is organized into working groups (WGs) with associated repositories, that focus on specific pieces of the ML platform. 
+## Kubeflow Components
+
+The Kubeflow ecosystem is composed of multiple open-source projects for each stage in
+[the ML lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
+
+Learn more about each project in [the Kubeflow documentation](https://www.kubeflow.org/docs/components/).
+
+Please use the following GitHub repositories to open issue and pull requests for different Kubeflow
+components:
+
+| Component                  | Source Code                                                                   |
+| -------------------------- | ----------------------------------------------------------------------------- |
+| KServe                     | [`kserve/kserve`](https://github.com/kserve/kserve)                           |
+| Kubeflow Katib             | [`kubeflow/katib`](https://github.com/kubeflow/katib)                         |
+| Kubeflow Model Registry    | [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry)       |
+| Kubeflow MPI Operator      | [`kubeflow/mpi-operator`](https://github.com/kubeflow/mpi-operator)           |
+| Kubeflow Notebooks         | [`kubeflow/notebooks`](https://github.com/kubeflow/notebooks)                 |
+| Kubeflow Pipelines         | [`kubeflow/pipelines`](https://github.com/kubeflow/pipelines)                 |
+| Kubeflow Spark Operator    | [`kubeflow/spark-operator`](https://github.com/kubeflow/spark-operator)       |
+| Kubeflow Training Operator | [`kubeflow/training-operator`](https://github.com/kubeflow/training-operator) |
+
+For Kubeflow Platform components:
 
-* [AutoML](https://github.com/kubeflow/community/tree/master/wg-automl)
-* [Deployment](https://github.com/kubeflow/community/tree/master/wg-deployment)
-* [Manifests](https://github.com/kubeflow/community/tree/master/wg-manifests)
-* [Notebooks](https://github.com/kubeflow/community/tree/master/wg-notebooks)
-* [Pipelines](https://github.com/kubeflow/community/tree/master/wg-pipelines)
-* [Serving](https://github.com/kubeflow/community/tree/master/wg-serving)
-* [Training](https://github.com/kubeflow/community/tree/master/wg-training)
+- Use [`kubeflow/manifests`](https://github.com/kubeflow/manifests) for Kubeflow Manifests.
+- Use [`kubeflow/dashboard`](https://github.com/kubeflow/dashboard) for Kubeflow Profile Controller,
+  Central Dashboard, CRUD Web Apps, PVC Viewer, PodDefault, and Access Management components.
```

Comment:
This is incorrect, only the following apps (see https://github.com/kubeflow/kubeflow/issues/7549) are going to be in `kubeflow/dashboard`:

- `Central Dashboard`
- `Profile Controller`
- `KFAM`
- `PodDefaults`

The CRUD Web Apps and PVC Viewer will live in `kubeflow/notebooks`.

Reply:
@thesuperzapper @kimwnasptd Why are we planning to maintain CRUD Web Apps in `kubeflow/notebooks` since it is used in almost all Kubeflow UIs (Katib, Models WebApp, Notebooks, PVC Viewer) ?

Reply:
@andreyvelich we discussed it a lot in https://github.com/kubeflow/kubeflow/issues/7549, but the gist of it is that the biggest usage of the CRUD code is in Kubeflow Notebooks. Also, stuff like the PVC Viewer is seen as "part of Kubeflow notebooks" so it will also be in `kubeflow/notebooks`.

Really all that matters is that the common CRUD code stays maintained, and its easiest for the Notebooks WG to do that in the `kubeflow/notebooks` repo.

Finally, the `kubeflow/dashboard` repo is for the things which make the dashboard work, it has no dependency on the CRUD code, so makes no sense for the CRUD code to live there.

Reply:
@thesuperzapper How did you conclude that the biggest usage of CRUD is in `kubeflow/notebooks` ?

The common web apps handle an ability to communicate between dashboard and other web apps: https://github.com/kubeflow/kubeflow/tree/2898d0f61edf193209c82cf673608cf97c247c3a/components/crud-web-apps/common#frontend.
When @kubeflow/arrikto initiated this component, the intention was to provide unify styles across all Kubeflow web apps. So users will get similar experience by using various Kubeflow UIs.

If we don't maintain it in the `dashboard` repository, how can we make sure that other UIs are functional and problems can be addressed ? 
For example, Katib UI CI is broken since April 19th by migrating to the latest Kubeflow commit version: https://github.com/kubeflow/katib/pull/2313
cc @orfeas-k

Reply:
It's not just the Katib web app, let's not forget about KServe web app, which makes extensive use of this common code.

@ederign this conversation is very relevant to our recent discussion. How and where we maintain this common code will influence the success of these various web apps. I think we should align the conversation and if you want to take the lead on defining broad best practices and recommendation, this may be a good place to start